ios/Podfile

```
@@ -95,74 +95,88 @@ post_install do |installer|
-# Silences warnings for [CP] phases w/o I/O lists AND removes Hermes script on CI
-post_integrate do |installer|
-  # All Pods *.xcodeproj projects CocoaPods generated
-  pods_projects =
-    if installer.respond_to?(:generated_projects) && installer.generated_projects.any?
-      installer.generated_projects
-    else
-      [installer.pods_project].compact  # fallback for older CocoaPods
+  # ---- Hermes Fix 2: Remove "Replace Hermes" script phases everywhere ----
+  forbidden_markers = [
+    'Replace Hermes for the right configuration, if needed',
+    'Replace Hermes'
```

RuboCop (1.76.1)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)